akka-projection-core/src/main/scala/akka/projection/javadsl/ProjectionManagement.scala

```diff
@@ -48,4 +48,31 @@ import akka.projection.scaladsl
   def clearOffset(projectionId: ProjectionId): CompletionStage[Done] =
     delegate.clearOffset(projectionId).toJava
 
+  /**
+   * Is the given Projection paused or not?
+   */
+  def isPaused(projectionId: ProjectionId): CompletionStage[java.lang.Boolean] =
+    delegate.isPaused(projectionId).map(java.lang.Boolean.valueOf).toJava
+
+  /**
+   * Pause the given Projection. Processing will be stopped.
+   * While the Projection is paused other management operations can be performed, such as
+   * [[ProjectionManagement.resume]].
```

Comment:
nitpicking..

we can resume after pausing, but the most interesting are the other operations: `clearOffset` and `updateOffset`

Reply:
That was my intention, thanks for spotting.